api/query/cache/service/app.yaml

```diff
@@ -0,0 +1,15 @@
+runtime: custom
+env: flex
+service: searchcache
+
+# Should match something at https://cloud.google.com/compute/docs/machine-types.
+resources:
+  # n1-standard-96 (TODO: Would prefer n1-highmem-96)
+  cpu: 96
+  memory_gb: 360
```

Comment:
Just a sanity check: you intend to use automatic scaling (by CPU utilization), right?

Reply:
Good point. We should probably manually scale to 2 instances to smooth over restarts.